docs/ui/components/Snippet/blocks/SnackInline.tsx

```diff
@@ -74,6 +97,8 @@ export const SnackInline = ({
     return code.replace(/%%placeholder-start%%.*%%placeholder-end%%/g, '');
   };
 
+  const codeLanguage = findPropInChildren(children as ReactElement, 'className')?.split('-')[1];
```

Comment:
```suggestion
  const prismBlockClassName = findPropInChildren(children as ReactElement, 'className');
  const codeLanguage = prismBlockClassName ? prismBlockClassName.split('-')[1] : 'jsx';
```

Nit, let's add a fallback to `jsx` if something goes wrong with the detection.